CoreBots.cs

```diff
@@ -1114,17 +1114,15 @@ public void RegisterQuests(params int[] questIDs)
                             // Finding the list of items you dont have yet.
                             List<SimpleReward> simpleRewards =
                                 kvp.Key.SimpleRewards.Where(r => r.Type == 2 &&
-                                    Bot.Inventory.GetQuantity(r.ID) <= 0 && Bot.Bank.GetQuantity(r.ID) <= 0).ToList();
```

Comment:
wasnt this put in place for ae being a fucking retard giving a house item 6 quantity... and the script wouldnt stop trying for it?

Reply:
*me reading the commit log changes* ic ight as long as it weorks